requirements.txt

```diff
@@ -1,18 +1,12 @@
-fontmake==1.3.0
 
+ufo2ft==0.6.2
 --find-links https://github.com/daltonmaag/vttLib/releases
-vttLib==0.7.4
+vttLib==0.7.6
 
 # Subrequirements by the above packages
 fonttools==3.20.1
 ufoLib==2.1.1
 defcon==0.3.5
 cu2qu==1.3.0
```

Comment:
also cu2qu should be optional (unless there's a global scope import cu2qu somewhere, haven't checked).
the outlines are already in truetype format, and you are passing convert convertCubics=False argument to compileTTF